tests/test_lms_api.py

```diff
@@ -126,7 +126,26 @@ def test_get_course_enrollment_not_found():
 
 
 @responses.activate
+def test_get_enrollment_course_modes():
+    course_id = "course-v1:edX+DemoX+Demo_Course"
+    response = {"course_modes": ['a', 'list', 'containing', 'course', 'modes']}
+    expected_return = ['a', 'list', 'containing', 'course', 'modes']
```

Comment:
@itsjeyd Oh sorry. This was part of the code I copied from the "Update course enrollment view" PR and I forgot to mark it. It landed on master [here](https://github.com/edx/edx-enterprise/commit/dedcf737b78592e21ca19d424b5521ddb05c8bb7#diff-82ee99218a64ea6e5fb77cb5c7db8fd3R112). Let me know if you still want me to apply the refactoring.